index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
```

Comment:
We have to fix the HTML validator error. I am using [this validator](https://validator.w3.org/#validate_by_upload)